bundles/binding/org.openhab.binding.dscalarm/META-INF/MANIFEST.MF

```diff
@@ -30,7 +30,9 @@ Import-Package: gnu.io,
  org.slf4j
 Service-Component: OSGI-INF/activebinding.xml, OSGI-INF/genericbindingprovider.xml
 Bundle-ClassPath: .
-Export-Package: org.openhab.binding.dscalarm
+Export-Package: org.openhab.binding.dscalarm,
+ org.openhab.binding.dscalarm.internal
+Bundle-ActivationPolicy: lazy
```

Comment:
Repeating what Kai said, you must not export internal packages.  In the Ecobee binding, I renamed my "internal.messages" package to "messages" so at least it's not considered internal.